components/ActivityDetails/index.js

```diff
@@ -36,14 +37,17 @@ export default function ActivityDetails({
           <StyledCategory key={category.id}>{category.name}</StyledCategory>
         ))}
       </StyledCateogriesUl>
-      <DeleteButton type="button" onClick={onDelete}>
+      <StyledButton type="button" onClick={onDelete}>
         Delete
-      </DeleteButton>
+      </StyledButton>
+      <Link href={`/activities/${activity.id}/update`}>
+        <StyledButton type="button">Update</StyledButton>
+      </Link>
```

Comment:
Please don't build 'Button Links'. Use either a button or a Link. In this case the Link is the better approach because you don't execute any logic.